content/projects/git-exercises/_index.md

```diff
@@ -403,4 +403,18 @@ Your repo should be all of the following:
 
 We just covered the basics here. Please make sure you understand this stuff. It's super important. Git might seem like a weird theoretical thing to a lot of you. It might seem completely unrelated to the actual job of writing code. But it's not. Git makes teamwork on dev teams possible. Without it we'd spend more time shouting at each other than writing useful code. So learn it. Be comfortable with it. When we start working in teams later on all will be made clear.
 
-If you are curious now, spend some time googling git branching strategies. We use the feature branching strategy here. We'll cover it in depth later on in the course.
\ No newline at end of file
+If you are curious now, spend some time googling git branching strategies. We use the feature branching strategy here. We'll cover it in depth later on in the course.
+
+## Instructions For Reviewers
+
+- Ensure that there are 3 branches: main, history, and milkshake-flavours
+- The main branch should consist of 4 files: .gitignore, README.md, history.txt, and milkshakes.md
+- The history branch should consist of 3 files: README.md, history.txt and milkshakes.md. Consider that main should be merged into history before the .gitignore file is created.
```

Comment:
Lets see if we can cut this down a bit, you can remove the specifics about what should be in history and milkshakes